src/components/Home.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Country from './Country';
+import './Home.css';
+import load from '../assets/loading.gif';
+
+function Home() {
+  const data = useSelector((state) => state.countries);
+  return (
+    <div className="home_container">
+
+      <div className="home_country_list">
+        {data.countries.length > 0
+          ? data.countries.map(
+            (item) => (
+              <Country
+                key={item.id}
+                id={item.id}
+                name={item.name}
+                regions={item.regions}
+                confirmed={item.today_confirmed}
+              />
+            ),
+          )
+          : <img src={load} alt="load" style={{ width: '40px', height: '40px' }} />}
+      </div>
+    </div>
+  );
+}
+
+export default Home;
```

Comment:
- [ ] Kindly note that One of the requirements says that ` When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.` Kindly provide A filter that can filter data .

Reply:
Hi @andersonlebon 
thanks for time you spent reviewing my work, i think this is optional